src/super_gradients/common/factories/datasets_factory.py

```diff
@@ -17,13 +17,10 @@ def __init__(self):
             "cifar_100": Cifar100DatasetInterface,
             "imagenet": ImageNetDatasetInterface,
             "tiny_imagenet": TinyImageNetDatasetInterface,
-            "coco2017_detection_v2": CocoDetectionDatasetInterfaceV2,
-            "coco2017_detection": CoCoDetectionDatasetInterface,
-            "coco2014_detection": CoCo2014DetectionDatasetInterface,
+            "coco2017_detection": CocoDetectionDatasetInterface,
```

Comment:
you are changing the string name... CoCoDetectionDatasetInterface > CocoDetectionDatasetInterface
this is an unnecessary breaking change...